lib/common/changer.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module ReleaseManager
+  module Common
+    class Changer
+      class << self
+        def change_promotion(request)
+          create_component
+          clone_ci_configs
+          git_helper.use_repo(CI_CONFIGS_DIR) do
+            modify_file(request)
+            commit_and_push_changes(request)
+          end
+        end
+
+        private
+
+        def create_component
+          @ci_job_configs = factory.create_component(file_name: CI_CONFIGS_DIR, url: CI_CONFIGS_URL, ref: '')
+        end
+
+        def clone_ci_configs
+          logger.info('Cloning ci-job-configs...')
+          cloner.clone_component(@ci_job_configs)
```

Comment:
Could just call create_component here, no need to store the result in a variable.